compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/jsx-outlining-simple.js

```diff
@@ -0,0 +1,36 @@
+// @enableJsxOutlining
+function Component({arr}) {
+  const x = useX();
+  return (
+    <>
+      {arr.map((i, id) => {
+        return (
+          <Bar key={id} x={x}>
+            <Baz i={i}></Baz>
```

Comment:
what if the same attribute name is used multiple times with different values (on different children)?

Reply:
ooh good call! changed to bailout for now and added a todo test case